packages/aws-cdk-migration/test/rewrite.test.ts

```diff
@@ -88,4 +86,94 @@ describe(rewriteImports, () => {
 
     console.log('Look! I did something!');`);
   });
+
+  test('correctly rewrites import namespaced Construct by moving to constructs', () => {
+    const output = rewriteImports(`
+  // something before
+  import * as cdk from '@aws-cdk/core';
+  // something after
+
+  ${constructFileBody('cdk.Construct')}
```

Comment:
I usually like to keep the test 'input', 'expectation' and 'actual' super simple and clear, even if the input has duplicates, so it's clear when the test fails what happened. 

As an example, if the contents returned by `constructFileBody()` is changed, it makes it hard to realize if all of the tests it uses should change or not.

Either way, minor comment. I'll let you take a call on this one.

Reply:
I agree in general and generally like to keep branching in tests to an absolute minimum. In this case, it's boilerplate that should always be the same across the tests it's used in so I felt comfortable extracting the repeated section.